modules/widgets/contact-info.php

```diff
@@ -123,6 +124,10 @@ function widget( $args, $instance ) {
 				}
 			}
 
+			if ( '' != $instance['email'] ) {
```

Comment:
I've added additional check and if the email is invalid, it will be displayed as plain text.